src/lib/OneClickDownloadButton.svelte

```diff
@@ -5,6 +5,8 @@
   export let mapId: string | undefined = undefined
   export let playlistUrl: string | undefined = undefined
 
+  export let fontSize: string = '1rem'
```

Comment:
Would it not make sense to have this be a set css style on the individual playlist page itself since you have styling related to this on there anyways?

Reply:
I think this is cleaner. Modifying a components styling from the parents component with `:global(...)` is always a bit iffy and can lead to hard to find and fix issues. The behavior becomes much less clear overall. I'd say it's best to avoid it - hence this solution. It's generally not great though - I wish there was a better way to handle such cases in Svelte and maybe there are and I just don't know these techniques/features.